examples/seismic/viscoelastic/wavesolver.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, model, geometry, space_order=4, **kwargs):
         self.geometry = geometry
 
         self.space_order = space_order
-        self.dt = self.model.critical_dt
+        self.dt = self.model.dtype(.9 * self.model.critical_dt)
```

Comment:
Not sure I like this:
1. We are seeing instabilities without this factor?
2. Why this way instead of modifying the critical dt calc?

Reply:
1. yes. You can look at Thorbecke's implementation where he basically says that with Q, it's not always stable and need to play with dt.
2. I can try to put in model, but that would probably make it smaller for some other examples too.

Reply:
1. Yeah, had seen that before.
2. Indeed, it's a bit tricky - I'm not sure if it's better to have our 'default' `critical_dt` air on the side of caution for 'robustness' or not? What do you think? But maybe we could add a `factor` `kwarg` which is by default 0.9 or 1.0 (don't really mind)? At least this way the api will robust/trivial to change for any tinkering down the line - and then people will easily be able to look at the docstring to understand why such factors are sometimes needed. But I think that some_factor*critical_dt with no comment should be avoided at least.